core-java-modules/core-java-8-2/src/test/java/com/baeldung/javaClass/ClassForNameAndInstanceUnitTest.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.javaClass;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class ClassForNameAndInstanceUnitTest {
+    ClassForNameAndInstance tester= new ClassForNameAndInstance();
+    @Test
+    public void givenString_whenValidClass_thenReturnClass() throws ClassNotFoundException {
```

Comment:
Tests don't need to throw specific exceptions just `throws Exception` would be fine. This will make the second test a lot cleaner to read too.